lib/jekyll/configuration.rb

```diff
@@ -175,7 +175,7 @@ def read_config_file(file)
       next_config
     rescue SystemCallError
       if @default_config_file ||= nil
-        Jekyll.logger.warn "Configuration file:", "none"
+        Jekyll.logger.warn "Configuration file:", "none (looking for '#{file}')"
```

Comment:
If the config file is not found, I'd presume that's probably an error, not
superfluous.  Is there a common, valid case when no config file is found?

On Mon, Aug 29, 2016 at 1:42 PM, Parker Moore <notifications@github.com>
wrote:

> In lib/jekyll/configuration.rb
> <https://github.com/jekyll/jekyll/pull/5300#discussion_r76651107>:
>
> > @@ -175,7 +175,7 @@ def read_config_file(file)
> >        next_config
> >      rescue SystemCallError
> >        if @default_config_file ||= nil
> > -        Jekyll.logger.warn "Configuration file:", "none"
> > +        Jekyll.logger.warn "Configuration file:", "none (looking for '#{file}')"
>
> I think the path should be in a debug message (when you use --verbose).
> It's superfluous for most, I think.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/jekyll/jekyll/pull/5300/files/afb717af9ce3b1f3067a145cbb0fd32794d868c7#r76651107>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABk5TfB8ER2hMVwCeUdwrlsTrkeuA4dks5qkxoPgaJpZM4JvW3o>
> .
>



-- 
    Mark Stosberg
    Senior Systems Engineer
    RideAmigos